lib/PhpParser/PrettyPrinter/Standard.php

```diff
@@ -449,6 +449,7 @@ protected function pExpr_Cast_Int(Cast\Int_ $node) {
 
     protected function pExpr_Cast_Double(Cast\Double $node) {
         $kind = $node->getAttribute('kind', Cast\Double::KIND_DOUBLE);
+        $cast = '';
```

Comment:
```suggestion
        /** @psalm-var Cast\Double::KIND_* $kind */
        $kind = $node->getAttribute('kind', Cast\Double::KIND_DOUBLE);
```

this will remove the issue about possibly undeclared `$cast`, tell psalm that `$cast` is always `non-empty-string`, and also help catch bugs when comparing it to other types :)

Reply:
Indeed, I knew way less about Psalm then, I could fix that better now :)

Thanks for the review, but I'm not sure @nikic is interested by improving type documentation in this repo. This PR has waited for months without answers, I kinda made my peace with that!